sql/telemetry_derived/error_aggregates/query.sql

```diff
@@ -0,0 +1,185 @@
+CREATE TEMP FUNCTION
+  udf_json_extract_int_map (input STRING) AS (ARRAY(
+    SELECT
+      STRUCT(CAST(SPLIT(entry, ':')[OFFSET(0)] AS INT64) AS key,
+             CAST(SPLIT(entry, ':')[OFFSET(1)] AS INT64) AS value)
+    FROM
+      UNNEST(SPLIT(REPLACE(TRIM(input, '{}'), '"', ''), ',')) AS entry
+    WHERE
+      LENGTH(entry) > 0 ));
+CREATE TEMP FUNCTION
+  udf_json_extract_histogram (input STRING) AS (STRUCT(
+    CAST(JSON_EXTRACT_SCALAR(input, '$.bucket_count') AS INT64) AS bucket_count,
+    CAST(JSON_EXTRACT_SCALAR(input, '$.histogram_type') AS INT64) AS histogram_type,
+    CAST(JSON_EXTRACT_SCALAR(input, '$.sum') AS INT64) AS `sum`,
+    ARRAY(
+      SELECT
+        CAST(bound AS INT64)
+      FROM
+        UNNEST(SPLIT(TRIM(JSON_EXTRACT(input, '$.range'), '[]'), ',')) AS bound) AS `range`,
+    udf_json_extract_int_map(JSON_EXTRACT(input, '$.values')) AS `values` ));
+CREATE TEMP FUNCTION udf_histogram_get_sum(histogram_list ANY TYPE, target_key STRING) AS (
+  (
+    SELECT
+      udf_json_extract_histogram(value).sum
+    FROM
+      UNNEST(histogram_list)
+    WHERE
+      key = target_key
+    LIMIT
+      1
+  )
+);
```

Comment:
I wonder if we should have some kind of udf for getting the sum of a histogram? Seems like something that would be useful in many places.

Reply:
`udf_json_extract_histogram(histogram).sum` is the same length as `udf_json_extract_histogram_sum(histogram)`, and because it's a sql UDF it's also just as efficient (@jklukas did an analysis to confirm this).